schema_based_on_diagram.sql

```diff
@@ -8,5 +8,28 @@ CREATE TABLE medical_histories (
     id SERIAL PRIMARY KEY,
     admitted_at TIMESTAMP,
     FOREIGN KEY patient_id INT REFERENCES patients(id),
-    status VARCHAR(80),
-);
\ No newline at end of file
+    status VARCHAR(80)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL(10,2) NOT NULL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    FOREIGN KEY medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments (
+    FOREIGN KEY id INT REFERENCES medical_histories(id),
+    type VARCHAR(50) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE invoice_items (
+    id SERIAL PRIMARY KEY,
+    unit_price DECIMAL(10,2) NOT NULL,
+    quantity INT NOT NULL,
+    total_price DECIMAL(10,2),
+    FOREIGN KEY invoice_id INT REFERENCES invoices(id),
+    FOREIGN KEY treatment_id INT REFERENCES treatments(id)
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all FOREIGN KEYS have indexes created.

Reply:
Hello @1GORDON. We have been checking the changes requested, and we ensured that all tables have their respective 'id' set as 'SERIAL PRIMARY KEY'. Is that what you referred to with 'indexes created'? Or should we 'CREATE INDEX' for future queries regarding the use of FOREIGN KEYS and PRIMARY KEYS?